tests/helpers/common/Chocolatey/Get-ChocolateyVersion.ps1

```diff
@@ -8,7 +8,7 @@
     [CmdletBinding()]
     param()
     if (-not $script:runningVersion) {
-        [NuGet.Versioning.NuGetVersion]$script:runningVersion = ((Invoke-Choco --version).Lines | Where-Object { $_ -NotMatch "please upgrade" }) -join '`r`n'
+        [NuGet.Versioning.NuGetVersion]$script:runningVersion = (Invoke-Choco --version).Lines | Where-Object { $_ -NotMatch "please upgrade" } | Select-Object -Last 1

```

Comment:
As you mentioned previously we only need the absolute last line so there is no longer a need for the Where-Object block AFAIK.
As such it can be changed to

```suggestion
        [NuGet.Versioning.NuGetVersion]$script:runningVersion = (Invoke-Choco --version).Lines | Select-Object -Last 1
```

or

```suggestion
        [NuGet.Versioning.NuGetVersion]$script:runningVersion = (Invoke-Choco --version).Lines | Where-Object { $_ -Match "^\d+\.[\d\.]+" } | Select-Object -First 1

```

Reply:
I'm not entirely sure where the `NotMatch` on `please upgrade` was added/needed for, but my testing shows that the version check with the Regex works, so I went with that one.